{
  "configurations": [
    {
      "nodes": [
        {
          "cpeMatch": [
            {
              "criteria": "cpe:2.3:a:ibm:cognos_analytics:11.2.0:*:*:*:*:*:*:*",
              "matchCriteriaId": "1AB1B390-838B-4572-ACA0-2CFFDDB45EB1",
              "vulnerable": true
            },
            {
              "criteria": "cpe:2.3:a:ibm:cognos_analytics:11.2.1:*:*:*:*:*:*:*",
              "matchCriteriaId": "D500E11C-4A99-460F-B16A-4DA5895149D5",
              "vulnerable": true
            },
            {
              "criteria": "cpe:2.3:a:ibm:cognos_analytics:11.2.2:*:*:*:*:*:*:*",
              "matchCriteriaId": "BC703EBB-A37C-465C-8F7C-3B64AB3A71E4",
              "vulnerable": true
            },
            {
              "criteria": "cpe:2.3:a:ibm:cognos_analytics:11.2.3:*:*:*:*:*:*:*",
              "matchCriteriaId": "8CA6708A-851A-458C-81CC-0AE78CB0F0C0",
              "vulnerable": true
            },
            {
              "criteria": "cpe:2.3:a:ibm:cognos_analytics:11.2.4:-:*:*:*:*:*:*",
              "matchCriteriaId": "A1D81212-AFFE-4A73-AAC1-E558973FC452",
              "vulnerable": true
            },
            {
              "criteria": "cpe:2.3:a:ibm:cognos_analytics:12.0.0:*:*:*:*:*:*:*",
              "matchCriteriaId": "210893AF-E67A-49C1-80FC-59A1F1C1B32F",
              "vulnerable": true
            },
            {
              "criteria": "cpe:2.3:a:ibm:cognos_analytics:12.0.1:*:*:*:*:*:*:*",
              "matchCriteriaId": "CFDD4A63-2F81-48C8-8400-E1BE15C8EA3D",
              "vulnerable": true
            },
            {
              "criteria": "cpe:2.3:a:ibm:cognos_analytics:12.0.2:*:*:*:*:*:*:*",
              "matchCriteriaId": "0AF83D3E-FB2F-4A73-A18B-F55CB98124D6",
              "vulnerable": true
            }
          ],
          "negate": false,
          "operator": "OR"
        }
      ]
    }
  ],
  "cveTags": [],
  "descriptions": [
    {
      "lang": "en",
      "value": "IBM Cognos Analytics 11.2.0, 11.2.1, 11.2.2, 11.2.3, 11.2.4, 12.0.0, 12.0.1, and 12.0.2 is vulnerable to improper certificate validation when using the IBM Planning Analytics Data Source Connection. This could allow an attacker to spoof a trusted entity by interfering in the communication path between IBM Planning Analytics server and IBM Cognos Analytics server.  IBM X-Force ID:  283364."
    },
    {
      "lang": "es",
      "value": "IBM Cognos Analytics 11.2.0, 11.2.1, 11.2.2, 11.2.3, 11.2.4, 12.0.0, 12.0.1 y 12.0.2 es vulnerable a una validación de certificado incorrecta cuando se utiliza la conexión de origen de datos de IBM Planning Analytics . Esto podría permitir a un atacante falsificar una entidad de confianza interfiriendo en la ruta de comunicación entre el servidor IBM Planning Analytics y el servidor IBM Cognos Analytics. ID de IBM X-Force: 283364."
    }
  ],
  "id": "CVE-2024-25053",
  "lastModified": "2024-08-01T18:20:26.953",
  "metrics": {
    "cvssMetricV31": [
      {
        "cvssData": {
          "attackComplexity": "HIGH",
          "attackVector": "NETWORK",
          "availabilityImpact": "NONE",
          "baseScore": 5.9,
          "baseSeverity": "MEDIUM",
          "confidentialityImpact": "NONE",
          "integrityImpact": "HIGH",
          "privilegesRequired": "NONE",
          "scope": "UNCHANGED",
          "userInteraction": "NONE",
          "vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:H/A:N",
          "version": "3.1"
        },
        "exploitabilityScore": 2.2,
        "impactScore": 3.6,
        "source": "nvd@nist.gov",
        "type": "Primary"
      },
      {
        "cvssData": {
          "attackComplexity": "HIGH",
          "attackVector": "NETWORK",
          "availabilityImpact": "NONE",
          "baseScore": 5.9,
          "baseSeverity": "MEDIUM",
          "confidentialityImpact": "NONE",
          "integrityImpact": "HIGH",
          "privilegesRequired": "NONE",
          "scope": "UNCHANGED",
          "userInteraction": "NONE",
          "vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:H/A:N",
          "version": "3.1"
        },
        "exploitabilityScore": 2.2,
        "impactScore": 3.6,
        "source": "psirt@us.ibm.com",
        "type": "Secondary"
      }
    ]
  },
  "published": "2024-06-28T19:15:04.933",
  "references": [
    {
      "source": "psirt@us.ibm.com",
      "tags": [
        "VDB Entry"
      ],
      "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/283364"
    },
    {
      "source": "psirt@us.ibm.com",
      "tags": [
        "Vendor Advisory"
      ],
      "url": "https://www.ibm.com/support/pages/node/7156941"
    }
  ],
  "sourceIdentifier": "psirt@us.ibm.com",
  "vulnStatus": "Analyzed",
  "weaknesses": [
    {
      "description": [
        {
          "lang": "en",
          "value": "CWE-295"
        }
      ],
      "source": "psirt@us.ibm.com",
      "type": "Primary"
    }
  ]
}